zlogger_kit/zlog.py

```
@@ -137,72 +137,84 @@ def _write_log(self, message: str, **kwargs) -> None:
         with open(log_file, "a", encoding="utf-8") as f:
             f.write(log_content)
 
-    def debug(self, message: str, error: Exception = None, **kwargs) -> None:
+    def debug(self, message: str, error: Exception = None, correlation_id: str | None = None, transaction_id: str | None = None, **kwargs) -> None:
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)